### ISSUE TITLE
bug: cargo wix sign panics: arg `no-capture`'s `ArgAction` should be one of `SetTrue`, `SetFalse` which should provide a default

### ISSUE THREAD

**Issue by roylaurie** [volks73/cargo-wix#291]
I'm guessing that this is a recent regression.

This is being caused by a missing `.action(ArgAction::SetTrue)` for the `.arg()` setup for *--nocapture* in the **sign** sub-command. There's also a missing `.arg()` declaration for **sign**'s *--package* parameter.

I've verified a fix and will send over a PR.

```
cargo-wix:main$ RUST_BACKTRACE=1 cargo run -- wix sign
    Finished `dev` profile [unoptimized + debuginfo] target(s) in 0.04s
     Running `target/debug/cargo-wix wix sign`
thread 'main' panicked at /home/roylaurie/.cargo/registry/src/index.crates.io-6f17d22bba15001f/clap_builder-4.4.14/src/parser/matches/arg_matches.rs:181:17:
arg `no-capture`'s `ArgAction` should be one of `SetTrue`, `SetFalse` which should provide a default
stack backtrace:
   0: rust_begin_unwind
             at /rustc/9b00956e56009bab2aa15d7bff10916599e3d6d6/library/std/src/panicking.rs:645:5
   1: core::panicking::panic_fmt
             at /rustc/9b00956e56009bab2aa15d7bff10916599e3d6d6/library/core/src/panicking.rs:72:14
   2: clap_builder::parser::matches::arg_matches::ArgMatches::get_flag::{{closure}}
             at /home/roylaurie/.cargo/registry/src/index.crates.io-6f17d22bba15001f/clap_builder-4.4.14/src/parser/matches/arg_matches.rs:181:17
   3: core::option::Option<T>::unwrap_or_else
             at /rustc/9b00956e56009bab2aa15d7bff10916599e3d6d6/library/core/src/option.rs:978:21
   4: clap_builder::parser::matches::arg_matches::ArgMatches::get_flag
             at /home/roylaurie/.cargo/registry/src/index.crates.io-6f17d22bba15001f/clap_builder-4.4.14/src/parser/matches/arg_matches.rs:178:10
   5: cargo_wix::main
             at ./src/main.rs:1943:34
   6: core::ops::function::FnOnce::call_once
             at /rustc/9b00956e56009bab2aa15d7bff10916599e3d6d6/library/core/src/ops/function.rs:250:5

```

**Comment by Hoverbear** [comment]
I bumped into this today, I see that the latest release does not include this fix. Is there a chance we'll see a release with it, or should I keep building off the main branch?

**Comment by volks73** [comment]
@Hoverbear I have published a new release, v0.3.9. It is available on crates.io.

**Comment by Hoverbear** [comment]
Thanks so much, @volks73 ! :)
